docs/fr/index.md

```diff
@@ -29,7 +29,7 @@ Cette documentation est divisée en plusieurs parties :
 
 * La [documentation utilisateurs](./users/02_First_steps.md) pour découvrir
   les fonctionnalités de FreshRSS.
-* La [documentation administrateurs](../en/admins/01_Installation.md) (en
+* La [documentation administrateurs](../en/admins/01_Index.html) (en
```

Comment:
In principle this file is auto-generated based on the POT file but I think po4a should also pick up on the reverse.

https://github.com/FreshRSS/FreshRSS/blob/2b26f9dcbde3cf2f2657b33cf85245577cbacce0/docs/i18n/freshrss.fr.po#L3914-L3921